### PR TITLE
Moneropedia entries into Norwegian (A–B)

### PR DESCRIPTION
**Pull request by MindWallet**
The translation has not yet been reviewed

### CODE REVIEW

File: _i18n/nb-no/resources/moneropedia/account.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+aunt knuckle italics moisture hawk thorn iris abort
+chlorine smog uphill glass aptitude nowhere sewage plywood
+dual relic fierce divers anvil nodes bubble cabin abort
+**********************************************************************
+Starting refresh...
+Refresh done, blocks received: 21939                            
+Balance: 0.000000000000, unlocked balance: 0.000000000000
+Background refresh thread started
+[lommebok 4B15Zj]: █
```
Comment (erciccione) [inline _i18n/nb-no/resources/moneropedia/account.md:65]: Better keep this part in english. It's not translated in the CLI.

File: _i18n/nb-no/resources/moneropedia/address-book.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (erciccione) [inline _i18n/nb-no/resources/moneropedia/address-book.md:1]: This moneropedia entry will be removed soon. see #1219

File: _i18n/nb-no/resources/moneropedia/account.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+---
+terms: ["konto", "kontoer", "lommebok", "lommebøker"]
+summary: "Tilsvarende en bankkonto og inneholder alle dine sendte og mottatte transaksjoner"
+---
+
```
Comment (erciccione) [inline _i18n/nb-no/resources/moneropedia/account.md:5]: Please add the snippet for the versioning system: `{% include disclaimer.html translated="yes" translationOutdated="no" %}`. It has to be added to **all** moneropedia entries. See the German translation: https://github.com/monero-project/monero-site/pull/1224/files

**Review by erciccione — CHANGES_REQUESTED** [review]
A few things:

- Make sure to keep the english terms as well in the front matter entry `terms`. For example `terms: ["luftvegg"]` should be `terms: ["airgap", "luftvegg"]`.

- Make sure you are using the latest version of the repository when translating. There have been a few changes lately.

- Please close #1256

File: _i18n/nb-no/resources/moneropedia/account.md (hunk 3 above)
Comment (MindWallet) [reply _i18n/nb-no/resources/moneropedia/account.md:5]: Done. Thanks for letting me know. I have updated my fork a couple of days ago, also, as I noticed my files where from my fork from July. I just need to update this pull (if that is possible)

Comment (erciccione) [reply _i18n/nb-no/resources/moneropedia/account.md:5]: It's not really a great practice, but you can close this and make a new one if it's better for you. I think there is a way to update a fork using github's user interface, but i'm not sure about how it's done. As i was saying in the chat, using git locally is more powerful.

**Pull request closed without merging**